src/lib/address_resolve/AddressResolve_DefaultImpl.cpp

```diff
@@ -245,6 +246,32 @@ void Resolver::Shutdown()
 
 void Resolver::OnOperationalNodeResolved(const Dnssd::ResolvedNodeData & nodeData)
 {
+    ResolveResult result;
+
+    auto resolutionData = nodeData.resolutionData;
+    result.address.SetPort(resolutionData.port);
+    result.address.SetInterface(resolutionData.interfaceId);
+    result.mrpRemoteConfig = resolutionData.GetRemoteMRPConfig();
+    result.supportsTcp     = resolutionData.supportsTcp;
+
+    // Copy all the IPs that has been found and sort them by score.
+    for (size_t i = 0; i < resolutionData.numIPs; i++)
+    {
+#if !INET_CONFIG_ENABLE_IPV4
+        if (!resolutionData.ipAddress[i].IsIPv6())
```

Comment:
can this actually happen? can we have a class IPAddress created with IPv4 when IPv4 is not supported?

Reply:
I see this pattern already exists in other places in the file ... so I guess it causes no harm.